config/form-content/bestuursorgaan/form.ttl

```diff
@@ -52,7 +52,8 @@ ext:form
         ext:bestuursorgaanClassificatieCodeF;
     sh:group ext:bestuursorgaanPG;
     form:initGenerator ext:bestuursorgaanG;
-    ext:label "bestuursorgaan" .
+    ext:label "bestuursorgaan" ;
+    ext:prefix besluit:Bestuursorgaan .
```

Comment:
since capitals are usually meant for types and this will be used as a prefix, i would go for besluit:bestuursorgaan (etc) so that the resulting uri will be besluit:bestuursorgaan/1 (not sure this is a legit way of writing this, but you know what i mean)